### PR TITLE
@grafana/e2e: flaky chart data workaround

### PR DESCRIPTION
**Pull request by stevenvachon**
See commit message(s).

### CODE REVIEW

**Review by ryantxu — APPROVED** [review]
Is this after a query is added?  if so yes, the only way to assert that the query is requested is to hit the refresh icon

**Comment by stevenvachon** [comment]
Before the query is added

File: packages/grafana-e2e/src/flows/configurePanel.ts [hunk 1]
```diff
@@ -127,15 +127,20 @@ export const configurePanel = (config: PartialAddPanelConfig | PartialEditPanelC
...
     e2e().wait(2000);
 
-    if (!isExplore) {
+    // There is no usable data when the query is empty,
+    // and Cypress had an issue with some plugins where the request wasn't noticed, despite occuring in manual tests
+    if (isEdit) {
       // Avoid cache flakiness (where @chartData isn't requested)
+      // @todo this may not be necessary anymore
       e2e()
         .get('.refresh-picker-buttons .btn')
```
Comment (hugohaggmark) [inline packages/grafana-e2e/src/flows/configurePanel.ts:136]: Friendly reminder to use selectors instead :)

Comment (stevenvachon) [reply packages/grafana-e2e/src/flows/configurePanel.ts:136]: I have not forgotten :)

**Pull request merged**